app/views/pages/_mobile_menu.html.erb

```diff
@@ -8,7 +8,12 @@
           <div class="bar2"></div>
           <div class="bar3"></div>
         </span>
-        <span id="mobile-navbar-brand" class="navbar-brand">See More</span>
+        <span id="mobile-navbar-brand" class="navbar-brand">
+         <% @page.categories.each do |i| %>
```

Comment:
You could simplify this with 
`<%= @categories.collect(&:label).join(" and ") %>`

Reply:
nice. done.

Reply:
I see this in app/helpers/application_helper.rb